pkgs/test/patch-shebangs/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, runCommand }:
+
+let
+  bad-shebang = stdenv.mkDerivation {
+    name         = "bad-shebang";
+    unpackPhase  = ":";
```

Comment:
I think it's better to do ```dontUnpack``` instead of this kind of weird trick.

Reply:
No that's not a thing AFAIK, this is a pretty common approach to skip unpacking and it's better than setting phases directly. But we have dont* variants of most other phases, it would make sense to add it.

Reply:
I know of the other `do*` and `dont*`s, but I thought there was none for `unpack`.

Reply:
> Lose your unpack phase with this one weird trick. Maintainers hate it!

Reply:
Yeah that's what I was thinking too :D